### PR TITLE
Create log in

### LINKED ISSUE #17: Create a Log In Form component
Image

### PR DESCRIPTION
**Pull request by oskarprzybylski23**
close #17 

Created a login form component that creates a new session.

### CODE REVIEW

File: src/components/LoginForm/LoginForm.tsx [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+		event.preventDefault();
+		const response = await fetch('http://localhost:3000/login', {
+			method: 'POST',
+			headers: {
+				'Content-Type': 'application/json',
+			},
+			body: JSON.stringify({ email, password }),
+		});
+
+		console.log(email, password);
```
Comment (halimahexe) [inline src/components/LoginForm/LoginForm.tsx]: Should probably remove this now you know it works

**Review by halimahexe — CHANGES_REQUESTED** [review]
LGTM just one small change pls

File: src/components/LoginForm/LoginForm.tsx (hunk 1 above)
Comment (oskarprzybylski23) [reply src/components/LoginForm/LoginForm.tsx]: cheers, removed.

**Review by halimahexe — APPROVED** [review]
LGTM

**Pull request merged**